docs/pbc/all/search/202410.0/base-shop/third-party-integrations/algolia/algolia-personalization-for-headless-frontends.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Algolia Personalization for headless frontends
+description: Find out how you can enable Algolia personalization in your Spryker shop based on headless approach (custom frontend or mobile application). 
+last_updated: Nov 24, 2024
+template: howto-guide-template
+---
+
+{% info_block infoBox "Info" %}
+
+Default Spryker installation supports Algolia personalization only for YVES frontend.
```

Comment:
What's a `default Spryker installation`?

Reply:
we can rephrase to "we have code for collecting events OOTB only for Yves", for custom headless frontends we don't have anything OOTB